docs/markdown/Releases/upgrade-tips.md

```diff
@@ -7,25 +7,24 @@ createdAt: "2020-05-16T22:53:24.499Z"
 updatedAt: "2022-01-13T04:03:33.172Z"
 ---
 > 📘 Reminder: change the `pants_version` to upgrade
-> 
+>
 > Change the `pants_version` option in the `[GLOBAL]` scope in your pants.toml to upgrade.
-> 
+>
 > You can see all releases at <https://pypi.org/project/pantsbuild.pants/#history>.
 
 Upgrade one minor release at a time
 -----------------------------------
 
 Per our [Deprecation policy](doc:deprecation-policy), deprecations must last a minimum of one minor release. For example, something may be deprecated in 2.1.0 and then removed in 2.2.0.
 
-This means that it is helpful to upgrade one minor release at a time so that you can see all deprecation warnings. 
+This means that it is helpful to upgrade one minor release at a time so that you can see all deprecation warnings.
 
 You do not need to land every upgrade into your organization—often, you will want to upgrade your organization multiple versions at a time, e.g. 2.1.0 to 2.4.0. But, when you are working on the upgrade locally, it is helpful to iterate one version at a time.
 
 First, see if Pants can automatically fix any safe deprecations for you:
 
 ```bash
-# You may want to use `--no-fmt` if your BUILD files are 
-# not already formatted by Black.
+# You may want to use `--no-fmt` if your BUILD files are already formatted.
```

Comment:
Nack. The idea here is "you only want deprecation fixers, not formatting"

So, you could clarify to use:

> To avoid unrelated formatting changes, you may want to use `--no-fmt`